docs/Parameters.md

```diff
@@ -287,20 +287,20 @@ Possible Values: null
 Default: www.example.org
 
 
-### skuCapacityDTU
+### mysqlPgresVcores
```

Comment:
Note that this file is auto generated using a one line script at the top of this file. Not sure if you used it or not. No worries if you didn't, just flagging for future reference in case you didn't.

Reply:
Cool, and just tried, but it's almost perfect: Had to add the preamble manually! Done anyway!